src/senaite/core/upgrade/v02_06_000.py

```diff
@@ -1641,6 +1641,42 @@ def migrate_attachmenttypes_to_dx(tool):
     logger.info("Convert AttachmentTypes to Dexterity [DONE]")
 
 
+@upgradestep(product, version)
+def move_dynamicanalysisspecs(tool):
+    """Move dynamic analysis specs to senaite package
+    """
+
+    # run required import steps
+    tool.runImportStepFromProfile(profile, "typeinfo")
+    tool.runImportStepFromProfile(profile, "workflow")
```

Comment:
Please remove

Reply:
yes, Commander. We've disputed is is required or not, and I left it there